src/components/Teams/tdata.js

```diff
@@ -0,0 +1,185 @@
+var organizers = [
```

Comment:
Use `const` as this will remain constant and there no mutations to be done via code.